_data/utilities.yml

```
@@ -433,6 +433,7 @@ websites:
     img: tmobile.png
     tfa:
       - sms
+      - totp
```

```suggestion
      - totp
      - hardware
```
Since "Fingerprint or FaceID" is available as well (i.e. biometric features), the `hardware` method should be included as well (which is what we currently map biometric 2FA to).